thirdparty/djvulibre/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ endif()
 
 if(NOT $ENV{WIN32})
     set(PATCH_CMD sh -c "${ISED} \"s|-lstdc++||g\" libtool")
+else()
+    set(PATCH_CMD sh -c "true")
```

Comment:
Seems to work, but I think it'd be easier to follow if you simply defined it first.

I.e., something like:

```
# Apparently CMake is scoped.
set(PATCH_CMD sh -c "true")

if(NOT $ENV{WIN32})
    set(PATCH_CMD sh -c "${ISED} \"s|-lstdc++||g\" libtool")
```